src/main/java/com/iexec/common/utils/FeignBuilder.java

```diff
@@ -0,0 +1,58 @@
+package com.iexec.common.utils;
+
+import feign.Feign;
+import feign.Logger;
+import feign.RequestTemplate;
+import feign.Response;
+import feign.codec.Encoder;
+import feign.codec.StringDecoder;
+import feign.jackson.JacksonDecoder;
+import feign.jackson.JacksonEncoder;
+import feign.slf4j.Slf4jLogger;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+
+/**
+ * Returns a Feign builder with shared configuration to create various Feign client instances.
+ * <p>
+ * The shared configuration is:
+ * <ul>
+ * <li>JacksonEncoder with overridden encode method to deal with strings
+ * <li>JacksonDecoder with overridden decode method to deal with strings
+ * <li>SLF4J logger enabled with FULL details. Logs are displayed by configuring the log level on a target interface logger.
+ * <li>application/json Content-Type header
+ * </ul>
+ */
+public class FeignBuilder {
+
+    private FeignBuilder() {};
+
+    public static Feign.Builder createBuilder() {
```

Comment:
```suggestion
    public static Feign.Builder createBuilder(Logger.Level loggerLevel) {
```

Reply:
I asked myself to do this like this when writing this class.
It brings several interesting design questions:
* What should be the default `Logger.Level` value when calling `createBuilder` ?
* Do we link this to SLF4J log level ? (For example, if slf4j level is `DEBUG,` feign log level is `FULL` by default, if slf4j log level is `INFO`, feign log level is `BASIC` by default)

What do you think ?

Reply:
BASIC by default without SLF4J binding is fine

Reply:
done in 27bc189

Reply:
Thanks!